src/content/configuration/output.md

```diff
@@ -196,7 +196,7 @@ Make sure to read the [Caching guide](/guides/caching) for details. There are mo
 
 Note this option is called filename but you are still allowed to use something like `"js/[name]/bundle.js"` to create a folder structure.
 
-Note this options does not affect output files for on-demand-loaded chunks. For these files the [`output.chunkFilename`](#output-chunkfilename) option is used. It also doesn't affect files created by loaders. For these files see loader options.
+Note this option does not affect output files for on-demand-loaded chunks. For these files the [`output.chunkFilename`](#output-chunkfilename) option is used. It also doesn't affect files created by loaders. For these files see loader options.
```

Comment:
@jaketodaro Should 'loader options' link to something?

Reply:
I'm not sure what it would link to? Did you have page in mind? Anyone can write a loader for any file type and support whatever options they want right? I'm not aware of any page that lists all of the loaders and the options for them.

Reply:
@jaketodaro I would just rephrase that bit as:

> Files created by loaders aren't affected, in this case you would have to try the specific loader's available options.